AutoRouteManager.php

```diff
@@ -50,11 +55,13 @@ class AutoRouteManager
     public function __construct(
         AdapterInterface $adapter,
         UriGeneratorInterface $uriGenerator,
-        DefunctRouteHandlerInterface $defunctRouteHandler
+        DefunctRouteHandlerInterface $defunctRouteHandler,
+        UriContextCollectionBuilder $collectionBuilder
```

Comment:
I don't see much problems with the idea of releasing 2.0 in a few weeks. This should however still be mentioned in the CHANGELOG (even when releasing it as 2.0) and I would also like to have an UPGRADE-2.0.md file explaining the changes.